ua-snap/iem-webapp
# Adds people credits page

Testing:

 - Go to the new Credits menu option!
 - Play with screen sizes; it may or may not be passable.  (If you can, just fix it in-place in this branch).
 - Check for any goofs with glyphs or other text malformations (fix)
